component/controller/SerialPortBridgeGeneric.cpp

```diff
@@ -64,11 +64,12 @@ int SerialPortBridgeGenericClass = core::RegisterObject("Send data (ex: force, d
 SerialPortBridgeGeneric::SerialPortBridgeGeneric()
     : d_port(initData(&d_port, "port", "Serial port name"))
     , d_baudRate(initData(&d_baudRate, "baudRate", "Transmission speed"))
-    , d_packetOut(initData(&d_packetOut, "sentData", "Data to send"))
-    , d_packetIn(initData(&d_packetIn, "receivedData", "Data received"))
+    , d_packetOut(initData(&d_packetOut, "sentData", "Data to send: the value 245 will be set at the beginning of the sent data as a header,\n"
+                           "enabling to implement a header research in the 'receiving' code, for synchronization purposes."))
+    , d_packetIn(initData(&d_packetIn, "receivedData", "Data received: vector of unsigned char, each entry should be an integer between 0 and 244."))
     , d_size(initData(&d_size,(int)0,"size","Size of the arrow to send. Use to check sentData size. \n"
                                             "Will return a warning if sentData size does not match this value."))
-    , d_precise(initData(&d_precise,false,"precise","If true, will send the data in the format [MSB,LSB]*2*size"))
+    , d_precise(initData(&d_precise,false,"precise","If true, will send the data in the format [245,[MSB,LSB]*2*size,246]"))
```

Comment:
Actually, the value 246 is supposed to be the header of the second half of the packet when the option "splitPacket" is activated. Then again, it's a minor bug, and in normal operation it wouldn't be seen (value just dropped if not in "splitPacket" mode). Other than that,